hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/DbDefinitionsServiceImpl.java

```diff
@@ -196,7 +197,10 @@ private void initFiles(String folder) {
                 initNotifiers(initFolder);
             }
         } catch (Exception e) {
-            msgLog.errorDatabaseException("Error initializing files. Msg: " + e.getMessage());
+            if (log.isDebugEnabled()) {
```

Comment:
But I am not using log.debugf.  I was just printing the stack trace.  I'm not sure if we want to do this here but more than once I found it valuable.  If you have a suggestion for how you want this code snippet to look let me know, I will change it.